clients/src/main/java/org/apache/kafka/clients/ClusterConnectionStates.java

```diff
@@ -68,7 +68,7 @@ public ClusterConnectionStates(long reconnectBackoffMs, long reconnectBackoffMax
     }
 
     /**
-     * Return true iff we can currently initiate a new connection. This will be the case if we are not
+     * Return true if we can currently initiate a new connection. This will be the case if we are not
```

Comment:
iff means if and only if. So, this seems correct?

Reply:
@junrao 
Oh really didn't know about that. Sorry for interrupt you and thank you for letting me know:) 
I will close this PR.